packages/auth/src/AuthProvider.tsx

```diff
@@ -178,10 +182,17 @@ export class AuthProvider extends React.Component<
     return authToken
   }
 
+  // Note: Used by apollo middleware to get a fresh token
+  // State isn't changed, to prevent infinite loops
+  // Remove this, and call getToken when AuthProviderState.authToken is removed
+  getFreshToken = async () => {
+    return this.rwClient.getToken()
+  }
```

Comment:
I think it might be completely safe to delete all of `getToken` and replace it with this, since the `setState` in `getToken` doesn't really do anything anymore.

Reply:
It doesn't if you use the redwood apollo provider, but was trying to maintain backwards compatibility. Happy to remove this if you think making it a breaking change is ok!

Reply:
Ah, I understand why you've kept it, since `authToken` might be used by someone, but maybe you can just set it on line 122.

Reply:
Ah, I understand why you kept it, since someone might be using it to grab the `authToken` from the context. Nevermind!

Reply:
You know what, let's remove it. We can leave a comment on that @deprecated field that tells people what to do instead.

Reply:
I think the code would be a bajillion times easier to read.

Reply:
I agree! I'll remove `getFreshToken` and just leave `getToken` without the state changes then. No changes to FetchConfigProvider right? Will do shortly

Reply:
Awesome, yup. NO changes to `FetchConfProv`